julia-package/brainflow/Artifacts.toml

```diff
@@ -1,7 +1,7 @@
 [brainflow]
-git-tree-sha1 = "1681bc5c836c0465e02c42dce71db9572d723b6e"
+git-tree-sha1 = "551bbeee43bcec215c6dab34727a9f87bac06b09"
```

Comment:
will it be the same after merge?

Reply:
Why do you expect it to change? It only depends on the 3.8.1 libs.

Reply:
If we go for a 3.8.2 release then we need to update it again

Reply:
I thought its git commit, if its sha of compiled libs then what is sha256 = "520b54a5d8bd9247bd975ccbb73347f46129592acc6b80853a144d95a223ec4a" a few lines below it? They are not the same